gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/util/policy/HttpRoutingAssertionSimplifier.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayexport.util.policy;
+
+import org.w3c.dom.Element;
+import javax.inject.Singleton;
+
+import static com.ca.apim.gateway.cagatewayconfig.util.policy.PolicyXMLElements.*;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.getSingleChildElement;
+
+/**
+ * Simplifier for the HTTP Routing Assertion.
+ */
+@Singleton
+public class HttpRoutingAssertionSimplifier implements PolicyAssertionSimplifier {
```

Comment:
We will need this for other Assertions as well which are storing TLS goids?

Reply:
@nikhilahuja yes, we will have to do it for all other assertions if required, in this PR i have covered it for HttpRouting assertion as it is our core assertion